docker-compose/db/postgres-setup.sql

```diff
@@ -0,0 +1 @@
+CREATE DATABASE identityvault;
```

Comment:
Okay - can you also change the default settings.yaml then. That will keep the names consistent throughout.